frontend/src/screens/learner.tsx

```diff
@@ -64,33 +66,74 @@ const LearnerDashboard = () => {
             <HighlightedStudies />
 
             <StudiesContainer />
-
+            <Affix position={{ bottom: 20, left: 20 }}>
+                <Transition transition="slide-up" mounted={isVisible}>
+                    {(transitionStyles) => (
+                        <Button
+                            onClick={() => scrollTo({ y: 0 })}
+                            style={transitionStyles}
+                            radius="xl"
+                            p={0}
+                            w={48}
+                            h={48}
+                            bg="#007bff"
+                        >
+                            <IconArrowUp size="1.5rem" color="#fff" />
```

Comment:
IconCircleArrowUpFilled should be used, the color should also be modified to match the design spec